syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
Yea.  I've seen it done multiple places in PyTorch and other libraries.  I think that's a good way of doing it when we have a bunch of functions and we wanna export the contents of a module without exporting the module itself.